File: src/main/java/no/nav/service/pensjon/regler/to/InntektsavkortningUforetrygdResponse.java
package no.nav.service.pensjon.regler.to;


import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.beregning2011.BeregningsresultatUforetrygd;

/**
 * @author Tatyana Lochehina
 */
public class InntektsavkortningUforetrygdResponse extends ServiceResponse {


    /**
     * Beregningsresultat med ny inntektsavkortning.
     */
    private BeregningsresultatUforetrygd beregningsresultatUforetrygd;

    /**
     * Pakkseddel
     */
    private Pakkseddel pakkseddel;


}

File: src/main/kotlin/no/nav/pensjon/regler/to/InntektsavkortningUforetrygdResponse.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning2011.BeregningsresultatUforetrygd

class InntektsavkortningUforetrygdResponse : ServiceResponse() {
    /**
     * Beregningsresultat med ny inntektsavkortning.
     */
    var beregningsresultatUforetrygd: BeregningsresultatUforetrygd? = null
}